cumulusci/cli/tests/test_cci.py

```diff
@@ -523,8 +523,9 @@ def test_org_browser(self, browser_open):
         config.keychain.set_org.assert_called_once_with(org_config)
 
     @mock.patch("cumulusci.cli.cci.CaptureSalesforceOAuth")
+    @mock.patch("cumulusci.cli.cci.OrgConfig")
```

Comment:
The coveralls report of test coverage (https://coveralls.io/builds/27073372) shows that there is a newly uncovered line in the load_userinfo method of OrgConfig. That is a consequence of changing this to mock out the entire OrgConfig object. I'd suggest just mocking the OrgConfig._load_userinfo method instead so that OrgConfig.load_userinfo will still run.

```suggestion
    @mock.patch("cumulusci.cli.cci.OrgConfig._load_userinfo")
```

(We aim to never decrease test coverage.)